src/components/asthma/components/AsthmaAirQualities/AsthmaAirQualities.tsx

```diff
@@ -0,0 +1,110 @@
+import { faGear } from '@fortawesome/free-solid-svg-icons';
+import language from '../../../../helpers/language';
+
+export interface AsthmaAirQualitiesProps {
+    previewState?: 'loading' | AsthmaAirQualitiesPreviewState;
+    editZipCodesSurveyName?: string;
+    airQualityUrl: string;
+    date?: Date;
+    innerRef?: React.Ref<HTMLDivElement>;
+}
+
+export default function (props: AsthmaAirQualitiesProps) {
+    const [loading, setLoading] = useState<boolean>(true);
+    let [homeAirQuality, setHomeAirQuality] = useState<AsthmaAirQuality>();
+    let [workAirQuality, setWorkAirQuality] = useState<AsthmaAirQuality>();
+
+    useInitializeView(() => {
+        setLoading(true);
+
+        if (props.previewState === 'loading') {
+            return;
+        }
+        if (props.previewState) {
+            setHomeAirQuality(previewData[props.previewState].homeAirQuality);
+            setWorkAirQuality(previewData[props.previewState].workAirQuality);
+            setLoading(false);
+            return;
+        }
+
+        asthmaDataService.loadParticipant().then(participant => {
+            let loadAirQualities = props.date ?
+                asthmaDataService.loadAirQualitiesForDate(props.date) :
+                asthmaDataService.loadAirQualitiesForControlStatus(participant.getHomeAirQualityZipCode(), participant.getWorkAirQualityZipCode());
+
+            loadAirQualities.then(airQualities => {
+                setHomeAirQuality(airQualities.find(aq => aq.type === 'home'));
+                setWorkAirQuality(airQualities.find(aq => aq.type === 'work'));
+                setLoading(false);
+            });
+        });
+    }, [], [props.previewState]);
+
+    const onSetup = (): void => {
+        if (props.previewState || !props.editZipCodesSurveyName) return;
+        MyDataHelps.startSurvey(props.editZipCodesSurveyName);
+    }
+
+    const getSetupComponent = (label: string) => {
+        return <UnstyledButton onClick={() => onSetup()}>
+            <div className="mdhui-asthma-air-qualities-setup">
+                <div className="mdhui-asthma-air-qualities-setup-button">{language('asthma-air-qualities-setup-button-text')}</div>
+                <div className="mdhui-asthma-air-qualities-setup-label">{label}</div>
+            </div>
+        </UnstyledButton>;
+    };
+
+    const onClick = (airQuality: AsthmaAirQuality): void => {
+        if (props.previewState) return;
+        if (airQuality.status === 'not-configured' && props.editZipCodesSurveyName) {
+            MyDataHelps.startSurvey(props.editZipCodesSurveyName);
+        } else {
+            MyDataHelps.openApplication(props.airQualityUrl, {modal: true});
+        }
+    };
+
+    return <div className="mdhui-asthma-air-qualities" ref={props.innerRef}>
+        <div className="mdhui-asthma-air-qualities-header">
+            <div className="mdhui-asthma-air-qualities-title">{language('asthma-air-qualities-title')}</div>
```

Comment:
I would probably just drop the title since you could just use a ```<Title>``` component

Reply:
Wait nm, you need a gear on the right

Reply:
Position of the gear seems slightly off, feels like it should be 16px from the right edge of the card?


![Screenshot 2024-02-07 at 5 25 14 PM](https://github.com/CareEvolution/MyDataHelpsUI/assets/1643171/e5a0e518-7baa-4093-a68b-7c2b718c89ed)

Reply:
The gear was not an issue when switching to use `<Title>`.

I also adjusted the position of the gear.  It was originally right-justified, but I moved it in because the focus the focus traversal on the larger hidden touch target looked odd (see images below).  That is probably of little concern though, so I moved it back to the right.

**Previous with focus:**
![aqi1](https://github.com/CareEvolution/MyDataHelpsUI/assets/5408603/5802322a-71d3-4f39-9323-6cf194477f9d)
**Current with focus:**
![aqi2](https://github.com/CareEvolution/MyDataHelpsUI/assets/5408603/e74de49e-1a7d-4807-b407-bfb163a0d6ce)
**Current without focus:**
![aqi3](https://github.com/CareEvolution/MyDataHelpsUI/assets/5408603/c13af5e1-7d5c-4140-b1fe-4185be0acf8c)